lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -48,9 +55,27 @@ def method_missing(meth, *args)
 
       return mod.send(meth.to_s, *args)
     end
+
+    action = meth.to_s.delete_prefix('cmd_')
+    if mod && mod.kind_of?(Msf::Module::HasActions) && mod.actions.map(&:name).map(&:downcase).include?(action)
```

Comment:
Calling `String#downcase` may times is expensive in Ruby since it creates an a whole extra string object complete with many memory allocations. There _probably_ will never be so many actions that it would be a huge deal here, but it is worth using the efficient style where possible.

```suggestion
    if mod && mod.kind_of?(Msf::Module::HasActions) && mod.actions.map(&:name).any? { |a| a.casecmp?(action) }
```